### PR TITLE
strongswan does not like the modules to be compiled as part of kernel…

### PR DESCRIPTION
**Pull request by srinibas-zededa**
…, make them loadable modules. aws operates the ipsec tunnels in ikev1 mode, enabled the conifiguration flags for them

### CODE REVIEW

File: pkg/zedctr/trigger [hunk 1]
```diff
@@ -1 +1 @@
-1528784313
```
Comment (rvs) [inline pkg/zedctr/trigger]: @srinibas-zededa in the future please make sure NOT to include this file in your commits.

File: pkg/strongswan/Dockerfile [hunk 2]
```diff
@@ -29,16 +29,16 @@ ENV CONFIGURE_OPTS --prefix=/usr \
...
-            --disable-sha1 \
-            --disable-sha2 \
-            --disable-static
+            --enable-hmac \
+            --enable-ikev1 \
+            --enable-md5 \
+            --enable-rc2 \
+            --enable-sha1 \
+            --enable-sha2 \
+            --enable-static
```
Comment (rvs) [inline pkg/strongswan/Dockerfile]: why are we changing all these settings?

File: pkg/kernel/kernel_config-4.9.x-x86_64 [hunk 3]
```diff
@@ -808,10 +808,10 @@ CONFIG_NET_IPVTI=m
...
 CONFIG_NET_FOU=y
 CONFIG_NET_FOU_IP_TUNNELS=y
-CONFIG_INET_AH=y
-CONFIG_INET_ESP=y
-CONFIG_INET_IPCOMP=y
-CONFIG_INET_XFRM_TUNNEL=y
+CONFIG_INET_AH=m
+CONFIG_INET_ESP=m
+CONFIG_INET_IPCOMP=m
+CONFIG_INET_XFRM_TUNNEL=m
```
Comment (rvs) [inline pkg/kernel/kernel_config-4.9.x-x86_64]: so what exactly is the problem with compiling them statically?

Comment (srinibas-zededa) [reply pkg/kernel/kernel_config-4.9.x-x86_64]: I am closing this pull request. These mod probe warning does not affect the functionality.

**Comment by srinibas-zededa** [comment]
closing this pull request.

**Pull request closed without merging**